### PR TITLE
Updates the readme with instructions

### PR DESCRIPTION
**Pull request by gmmeyer**
Adds building and running instructions to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+You will need to set an API Key in your environment to enable the buildpack:
+
+```shell
+# set the environment variable
+cf set-env $YOUR_APP_NAME DD_API_KEY $YOUR_DATADOG_API_KEY
+# restage the application to get it to pick up the new environment variable and use the buildpack
+cf restage $YOUR_APP_NAME
+```
+
+### Statsd Away!
```
Comment (irabinovitch) [inline README.md]: Dogstatsd? or Statsd?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
-# Datadog CloudFoundry Buildpack
+# Datadog Cloud Foundry Buildpack
 
-This buildpack installs Dogstatsd into the container your app is running on.
+This is a [decorator buildpack](https://github.com/cf-platform-eng/meta-buildpack/blob/master/README.md#decorators) for Cloud Foundry. It will install a Datadog Dogstatsd binary on the container your app is running on. It uses the new Agent 6 build of Dogstatsd, which is very light weight and powerful. You should barely notice it running on your container.
```
Comment (irabinovitch) [inline README.md]: DogStatsD

Comment (irabinovitch) [reply README.md]: in the container or on the container?

Comment (irabinovitch) [reply README.md]: I'd probably cut this "It uses the new Agent 6 build of Dogstatsd, which is very light weight and powerful. You should barely notice it running on your container."

Maybe just . "This only contains the DogStatsd components of Datadog Agent, which as limited to no overhead."

File: README.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+
+## Use
+
+### Install the [Meta Buildpack](https://github.com/cf-platform-eng/meta-buildpack#how-to-install-the-meta-buildpack)
+First, you will have to [install the Meta Buildpack](https://github.com/cf-platform-eng/meta-buildpack#how-to-install-the-meta-buildpack). This enables apps to use decorator buildpacks. Follow the instructions to get the buildpack and upload it if you don't already have it.
+
+### Upload the Datadog Cloud Foundry Buildpack
+You can get it from the [releases page](https://github.com/DataDog/datadog-cloudfoundry-buildpack/releases) here, or you can just [download the latest version directly from us](https://cloudfoundry.datadoghq.com/datadog-cloudfoundry-buildpack/datadog-cloudfoundry-buildpack-latest.zip). After you download the zipfile, you will have to upload it to Cloud Foundry:
```
Comment (irabinovitch) [inline README.md]: Do we need to offer both links? Should we just point at Datadog?

Download the latest Datadog [build pack release]((https://cloudfoundry.datadoghq.com/datadog-cloudfoundry-buildpack/datadog-cloudfoundry-buildpack-latest.zip). After you downloading the zipfile, you will have to upload it to your Cloud Foundry environment.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,32 @@
...
+
+```shell
+# set the environment variable
+cf set-env $YOUR_APP_NAME DD_API_KEY $YOUR_DATADOG_API_KEY
+# restage the application to get it to pick up the new environment variable and use the buildpack
+cf restage $YOUR_APP_NAME
+```
+
+### Statsd Away!
+You're all set up to use Dogstatsd. Import the relevant library and start sending data! To learn more, [check our our documentation](https://docs.datadoghq.com/guides/dogstatsd/).
```
Comment (irabinovitch) [inline README.md]: DogStatsD

Comment (irabinovitch) [reply README.md]: Should we link this to our library page:
docs.datadoghq.com/libraries/

File: README.md (hunk 2 above)
Comment (gmmeyer) [reply README.md]: what's the difference between in and on?

File: README.md (hunk 4 above)
Comment (gmmeyer) [reply README.md]: Yes I think so, I'll do that!

**Pull request merged**